actor/src/main/scala/org/apache/pekko/serialization/Serialization.scala

```diff
@@ -457,8 +453,12 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
     }
   }
 
-  private def warnUnexpectedNonAkkaSerializer(clazz: Class[_], ser: Serializer): Boolean = {
-    if (clazz.getName.startsWith("org.apache.pekko.") && !ser.getClass.getName.startsWith("org.apache.pekko.")) {
+  @nowarn("msg=deprecated")
+  private def warnUnexpectedNonPekkoSerializer(clazz: Class[_], ser: Serializer): Boolean = {
+    import scala.collection.JavaConverters._ // switch to scala.jdk.CollectionConverters once Scala 2.12 support is dropped
+    if (clazz.getName.startsWith("org.apache.pekko.") && !ser.getClass.getName.startsWith("org.apache.pekko.") &&
```

Comment:
extract `"org.apache.pekko.` to a constant field?

Reply:
This might make sense in a separate PR, at least when I was renaming the package to `org.apache.pekko` there were a few cases where package names was being checked `clazz.getName`?